data/hlint.yaml

```diff
@@ -247,6 +247,8 @@
 
     # TRAVERSABLES
 
+    - warn: {lhs: traverse pure x, rhs: pure x, name: "Traversable law"}
+    - warn: {lhs: traverse (pure . f) x, rhs: pure (map f x), name: "Traversable law"}
```

Comment:
Is this one true? What if the type isn't a list, but a different foldable structure?

Reply:
It definitely feels like it would be sensible for this to be right, but I'm not so sure anymore.

Reply:
Maybe this is true for `fmap` rather than `map`?

Reply:
Certainly it would be with `fmap`, yeah, but I'm not sure even that's true. I couldn't figure out a proof last time so I just ditched it for now..

Reply:
I'm confident enough in `traverse (pure . f) x` ==> `pure (fmap f x)` that I'm happy to include it.

Reply:
Changed it